app/models/project.rb

```diff
@@ -1,4 +1,12 @@
 class Project <ApplicationRecord
   validates_presence_of :name, :material
   belongs_to :challenge
+
+  has_many :contestant_projects
+  has_many :contestants, through: :contestant_projects
+
+  def average_contestant_experience
```

Comment:
Look into using ActiveRecord's `average` function here instead.

Reply:
Thanks! I also was not using .sum right, haha.

```ruby
total_experience = contestants.sum(&:years_of_experience).to_f
```